content/How to Make My First Gem - Timothy Peraza.md

```diff
@@ -0,0 +1,128 @@
+# **How to Make My First Gem**
+
+Every developer, who has worked with ruby on rails and has felt pasion or intenrest for the open source world, has wanted or wants to do a contribution to this community of development with a gem which could solves a particular or generic problem or improve existing one, that appears in differents projects across the developer carrer. 
+
+However, sometimes happen that one doesn't know where to start or how is a gem made or other times one thinks or has the idea that to make a gem is a really hard or complecated task. For that reason, in this articule is searched to inquire how to make your first gem and to explain a bit the different component of its.
+
+Here at Rootstrap, we are much interested to be reconized in the open source world. For that, from a time, we have been working in this topic, making libraries for diferents lenguajes and frameworks and generating visible with blogs and posts in social media. In particular, we built [rsgem](https://github.com/rootstrap/rsgem), a gem which generates a base project to make a gem in which one this article is based on.
+
+
+## Stargint the Gem
+
+The first step to start is to install the rsgem, this is able to do easily using the command gem install rsgem. After we have the gem installed, we have to run the command rsgem new name, with the name that we wante for our gem. In particular, in this blog, we run the command rsgem new blog_gem, how you see this will generate a project with the next structure
```

Comment:
```suggestion
The first step to start is to install the rsgem, this is able to do easily by running `$ gem install rsgem`. After we have the gem installed, we have to run `$ rsgem new [name]`, with the name that we want for our gem. In particular, in this blog, we will be running the command `$ rsgem new blog_gem`, and this will generate a project with the following structure:
```